src/frontend/src/pages/ProjectDetailPage/ProjectEdit/ProjectEditContainer.tsx

```diff
@@ -209,10 +209,11 @@ const ProjectEditContainer: React.FC<ProjectEditContainerProps> = ({ project, ex
             </FormControl>
           </Grid>
         </PageBlock>
-        <PageBlock title="Links">
-          <LinksEditView watch={watch} ls={links} register={register} append={appendLink} remove={removeLink} />
-        </PageBlock>
         <Stack spacing={4}>
+          <Box>
+            <Typography variant="h5">{'Links'}</Typography>
```

Comment:
I would say add a bit of space between the links title and the rest of the content (similar to the space in the page block for project summary rn) otherwise looks good

Reply:
also merge conflicts